data/mods/CrazyCataclysm/crazy_speech.json

```diff
@@ -250,5 +250,45 @@
     "speaker": "aperture_potato",
     "sound": "Are you even listening to me?",
     "volume": 20
+  },
+  {
+	"type": "snippet",
```

Comment:
24 instances of "Tab" whitespace instead of four spaces. Starting here and ending line 292.